kolibri/plugins/learn/assets/src/views/content-card-carousel/index.vue

```diff
@@ -92,7 +121,14 @@
       },
       header: { type: String },
       subheader: { type: String },
-      genLink: {
+      showViewMore: { type: Boolean }, // IDEA collapse into viewMorePageLink
+      viewMorePageLink: {
+        type: Object,
+        validator(pageLink) {
+          return validateLinkObject(pageLink);
+        },
+      },
+      genContentLink: {
```

Comment:
Had to get specific about links because we also need to pass in the `viewMorePageLink`